pkg/output/table.go

```diff
@@ -13,31 +15,85 @@ type tableManager struct {
 	isWide bool
 }
 
-func (c *tableManager) Output(ctx context.Context, out interface{}) error {
-	var header func() []string
-	var rows func() [][]string
+func (c *tableManager) isInt(v string) bool {
+	_, err := strconv.Atoi(v)
+	return err == nil
+}
+
+func (c *tableManager) isIntColumn(tableData [][]string, col int) bool {
+	for _, row := range tableData {
+		if row[col] != "" || !c.isInt(row[col]) {
```

Comment:
This logic doesn't work. It has to be `row[col] != "" && !c.isInt(row[col])`

Reply:
If it's '3' for example, we're already passing the first check and return false incorrectly